.github/workflows/builds.yml

```
@@ -309,6 +309,8 @@ jobs:
         uses: pnpm/action-setup@v2.0.1
         with:
           version: 6.24.1
+      - name: Install node-gyp
+        run: npm install -g node-gyp
```

I would strongly prefer that we specify a version. Looking at [this line](https://github.com/ferdium/ferdium-app/blob/develop/package.json#L186) - we should go with `8.4.1`

---

Oops of course, forgot :facepalm: 
```suggestion
        run: npm install -g node-gyp@8.4.1
```